auth/server/src/main/java/com/seagates3/policy/BucketPolicyAuthorizer.java

```diff
@@ -63,6 +65,40 @@
         serverResponse =
             authorizeOperation(requestBody, requestedOperation, requestor);
       }
+      // Below authorization is for CopyObject in case of secondary operations
+      if ((serverResponse == null ||
```

Comment:
Authorizing actions present in S3ActionList even if someone passes it accidently.
Not putting specific check of PutObject here. Earlrier it was added but then removed per @bkirunge7  review comment.